addons/web/static/src/views/fields/properties/property_value.js

```diff
@@ -200,15 +200,15 @@ export class PropertyValue extends Component {
             newValue = newValue && serializeDate(newValue);
         } else if (this.props.type === "integer") {
             try {
-                newValue = parseInteger(newValue) || 0;
+                newValue = newValue === "" ? false : parseInteger(newValue);
```

Comment:
is `newValue === "" ? false` correct ?
with normal field if you empty the <input/> is set to value to 0.00, not to false
See https://drive.google.com/file/d/1b9maXuvASMQutJ7V1t9hwRyg0_0_1jAO/view?usp=sharing